1-the-basics/css/styles.css

```
@@ -0,0 +1,70 @@
+header{
+    background-color: aqua;
+}
+
+footer{
+    background-color: bisque;
+}
+.bg-img{
+    padding-top: 10px;
+    background-image: url('https://upload.wikimedia.org/wikipedia/commons/thumb/e/ef/Coat_of_arms_of_Colombia.svg/1200px-Coat_of_arms_of_Colombia.svg.png');
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+.myFont {
+    font-family: system-ui;
+    font-size: 18px;
+}
+.form-mg {
+    width: 70%;
+    margin-left: 15%;
+}
+
+.resalt:hover{
+    background: rgb(254,221,0);
+    background: linear-gradient(180deg, rgba(254,221,0,0.7483368347338936) 100%, rgba(255,255,255,1) 100%);
+    color: black;
+    padding:10px;
```

```suggestion
    padding: 10px;
```
be consistent with your spaces.